website/blog/2019-09-05-7.6.0.md

```diff
@@ -0,0 +1,98 @@
+---
+layout: post
+title:  "7.6.0 Released: Private static accessors and V8 intrinsic syntax"
+author: Nicolò Ribaudo
+authorURL: https://twitter.com/NicoloRibaudo
+date:   2019-09-06 0:00:00
+categories: announcements
+share_text: "Babel 7.6.0 Released"
+---
+
+We just released a new minor Babel version!
+
+It includes support for static private accessors in classes, and parser support for the V8 intrinsics syntax. We also fixed a bunch of long-stansing issues related to TDZ handling, and improved support for do-expressions. You can read the whole changelog [on GitHub](https://github.com/babel/babel/releases/tag/v7.6.0).
+
+<!-- truncate -->
+
+Thanks to [Serhii Muryhin](https://github.com/skyiea), [Ashwin Ramaswami](https://github.com/epicfaace), [Sungmin Lee](https://github.com/ifsnow), [Serge Havas](https://github.com/Sinewyk), [Ziad El Khoury Hanna](https://github.com/zaygraveyard), [Even Alander](https://github.com/deificx), [Shrey Banga](https://github.com/banga), and [Dylan Kirkby](https://github.com/dsgkirkby) for their first PRs! (And also thanks to [Codetriage README Bot](https://github.com/codetriage-readme-bot) and [dependabot](https://help.github.com/en/articles/configuring-automated-security-fixes) 🐸)
```

Comment:
The use of the “🐸” here might not be ideal given its current political associations. I’d suggest to just remove it or use another emoji.

Reply:
Thank you, I wasn't aware of it.

Reply:
```suggestion
Thanks to [Serhii Muryhin](https://github.com/skyiea), [Ashwin Ramaswami](https://github.com/epicfaace), [Sungmin Lee](https://github.com/ifsnow), [Serge Havas](https://github.com/Sinewyk), [Ziad El Khoury Hanna](https://github.com/zaygraveyard), [Even Alander](https://github.com/deificx), [Shrey Banga](https://github.com/banga), and [Dylan Kirkby](https://github.com/dsgkirkby) for their first PRs! (And also thanks to [Codetriage README Bot](https://github.com/codetriage-readme-bot) and [dependabot](https://help.github.com/en/articles/configuring-automated-security-fixes) 🤖)
```